From 7cd5cff42bfab1914c9895876e6c6708f30c6db7 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Wed, 15 Mar 2023 08:50:41 +0200
Subject: [PATCH] [SW-136144] gaudi3: include specs_root_h9 temporarily

As a preparation to remove SPECS_H9_DIR. This patch adds
support to include SPECS_ROOT_H9 to get the specs patch
https://gerrit.habana-labs.com/#/c/310879/ pass in CI
for host application and embedded firmware.

This patch should be reverted once SPECS_DIR is set to
SPECS_ROOT_H9 for gaudi3.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I4756929603a579887a6f55ee21e9696c1cf6bb9b
---

diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index 2b7999e..d1e511c 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -14,7 +14,7 @@
 ZEPHYR_CONF_OPTS += -DEXTRA_CFLAGS=-Werror -Wno-misleading-indentation
 ZEPHYR_CONF_OPTS += -DBUILD_VERSION="$(shell cut -d- -f2- $(ZEPHYR_BUILDDIR)/localversion-habanalabs)"
 ZEPHYR_CONF_OPTS += -DBUILD_MACHINE="$(shell hostname -s)"
-ZEPHYR_CONF_OPTS += -DCMAKE_C_FLAGS="-I$(SPECS_EMBEDDED_DIR) -I$(SPECS_EMBEDDED_DIR)/$(DEV_PREFIX) -I$(SPECS_ROOT) -I$(SPECS_EXT_ROOT) -I$(SPECS_ROOT)/$(DEV_PREFIX)/asic_reg -I $(SPECS_H9_DIR)"
+ZEPHYR_CONF_OPTS += -DCMAKE_C_FLAGS="-I$(SPECS_EMBEDDED_DIR) -I$(SPECS_EMBEDDED_DIR)/$(DEV_PREFIX) -I$(SPECS_ROOT) -I$(SPECS_EXT_ROOT) -I$(SPECS_ROOT)/$(DEV_PREFIX)/asic_reg -I $(SPECS_H9_DIR) -I$(SPECS_ROOT_H9)"
 ZEPHYR_CONF_OPTS += -DDTS_EXTRA_CPPFLAGS="-I$(SPECS_EMBEDDED_DIR)"
 
 DEVICE_NAME = $(firstword $(subst _, ,$(HL_DEV)))
